config/application.rb

```diff
@@ -118,5 +115,21 @@ class Application < Rails::Application
     config.site_version = "2.10.0"
 
     config.lti_settings = Rails.application.config_for(:lti_settings)
+
+    config.action_controller.default_url_options = { }
+    config.action_mailer.default_url_options = {}
+
+    # Set application host for mailer
+    config.action_mailer.default_url_options = { host: ENV['MAILER_HOST'] }
+
+    # Configure the host and port of generated urls
+    if !ENV['DEFAULT_URL_HOST'].empty? 
```

Comment:
Empty does not work when it's a nilclass
` undefined method `empty?' for nil:NilClass (NoMethodError)`

Reply:
Fixed.